pages/common/git-grep.md

```diff
@@ -0,0 +1,19 @@
+# git-grep
+
+> Find strings inside files as they existed anywhere in repo history. Accepts a lot of the same flags as regular grep. Learn more here: https://git-scm.com/docs/git-grep.
+
+- Search for a string in working directory:
+
+`git grep {{search_string}}`
+
+- Search for string in files matching a pattern in working directory:
```

Comment:
Perhaps this will read better?

```suggestion
- Search for string in files matching a pattern in the current working directory:
```

Reply:
Again, "current working directory" is misleading. Maybe just say "Search for a string in files matching a pattern:"